rules.tf

```
@@ -108,6 +108,15 @@ resource "aws_wafv2_web_acl" "default" {
     sampled_requests_enabled   = lookup(var.visibility_config, "sampled_requests_enabled", true)
   }
 
+  dynamic "custom_response_body" {
+    for_each = var.custom_response_body
+    content {
+      content      = lookup(var.custom_response_body, "content", null)
+      content_type = lookup(var.custom_response_body, "content_type", null)
+      key          = lookup(var.custom_response_body, "key", null)
```

Since all parameters are required if the map has elements, wouldn't this be better?

```suggestion
      content      = custom_response_body.key["content"]
      content_type = custom_response_body.key["content_type"]
      key          = custom_response_body.key["key"]
```

---

Thnaks 😄